### PR TITLE
DRAFT Get products route

### PR DESCRIPTION
**Pull request by diddydeeable**
not finished yet but you can merge it if you need it , please don't delete the branch as I am still working on it @AlexPD93

### CODE REVIEW

File: model/product.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    products.product_description,
+    products.category,
+    products.price,
+    products.product_image,
+    products.stock
+    FROM products 
+WHERE products.product_name = ?
+`);
+export function getProductByName(name: string) {
+    return select_product_by_name(name);
```
Comment (AlexPD93) [inline model/product.ts:65]: Do we want to get all books that have that name?

**Review by AlexPD93 — COMMENTED** [review]
Should we replace products everywhere with books so it is less confusing?

File: model/product.ts (hunk 1 above)
Comment (diddydeeable) [reply model/product.ts:65]: it's a variable

Comment (diddydeeable) [reply model/product.ts:65]: so that's why if it was a dynamic route name would be replaced by the actual name of the book we want

**Pull request closed without merging**